### ISSUE TITLE
RMS compile issue during installation

### ISSUE THREAD

**Issue by calvinp0** [ReactionMechanismGenerator/RMG-Py#2353]
### Bug Description
As of late, when a new user attempts to install RMG-Py (assuming they are following the developer instructions), they are running into a compiling issue with Julia/RMS. The error is along the lines of Julia not being able to find matplotlib from pyimport. Thus, RMS is not properly installed and RMG-Py cannot be run successfully. 

### How To Reproduce
1. On a clean machine, follow the developer installation instructions as written [here](https://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/anacondaDeveloper.html)
2. During the installing of the link between Python and Julia, a compile error occurs

### Expected Behavior
It is expected that RMS compiles successfully.

### Installation Information
Describe your installation method and system information.
 - OS (include version if known): Ubuntu 22.04
 - Installation method: Installation from source, using Mambaforge
 - RMG version information: 3.1.0-620-g2cdc2ea7c
   
   - RMG-Py: 3.1.0-620-g2cdc2ea7c
   - RMG-database: 3.0.0-584-gbd1319070

### Additional Context
The solution I am using for T3, as it also install RMG-Py is as follows:
```
#Prior to activation of the environment, the LD_LIBRARY_PATH needs to be set as an environmnet variable when rmg_env is activated.
#This exporting and unsetting will solve the RMS installation during the Julia compile and also when the environment is deactivated
#the original LD_LIBRARY_PATH is set.
echo "export OLD_LD_LIBRARY_PATH=${LD_LIBRARY_PATH}" >> $CONDA_BASE/envs/rmg_env/etc/conda/activate.d/env_vars.sh
echo "export LD_LIBRARY_PATH=$CONDA_BASE/envs/rmg_env/lib:$LD_LIBRARY_PATH" >> $CONDA_BASE/envs/rmg_env/etc/conda/activate.d/env_vars.sh
echo "export LD_LIBRARY_PATH=${OLD_LD_LIBRARY_PATH}" >> $CONDA_BASE/envs/rmg_env/etc/conda/deactivate.d/env_vars.sh
echo "unset OLD_LD_LIBRARY_PATH" >> $CONDA_BASE/envs/rmg_env/etc/conda/deactivate.d/env_vars.sh
```

**Comment by JacksonBurns** [comment]
I have attempted this install on Ubuntu 20 and had the same issue - here is a snippet of the error
```
ERROR: LoadError: LoadError: InitError: PyError (PyImport_ImportModule

The Python package matplotlib could not be imported by pyimport. Usually this means
that you did not install matplotlib in the Python version being used by PyCall.

PyCall is currently configured to use the Python version at:

/home/jackson/mambaforge/envs/rmg/bin/python

and you should use whatever mechanism you usually use (apt-get, pip, conda,
etcetera) to install the Python package containing the matplotlib module.

One alternative is to re-configure PyCall to use a different Python
version on your system: set ENV["PYTHON"] to the path/name of the python
executable you want to use, run Pkg.build("PyCall"), and re-launch Julia.

Another alternative is to configure PyCall to use a Julia-specific Python
distribution via the Conda.jl package (which installs a private Anaconda
Python distribution), which has the advantage that packages can be installed
and kept up-to-date via Julia.  As explained in the PyCall documentation,
set ENV["PYTHON"]="", run Pkg.build("PyCall"), and re-launch Julia. Then,
To install the matplotlib module, you can use `pyimport_conda("matplotlib", PKG)`,
where PKG is the Anaconda package that contains the module matplotlib,
or alternatively you can use the Conda package directly (via
`using Conda` followed by `Conda.add` etcetera).
```
`matplotlib` is, in fact, installed and recognized by the environment. Both a `pip install matplotlib` and `mamba install matplotlib` state that it is already present.

Following the MacOS install instructions, the compilation for RMS was fine and RMG seems to work normally.

**Comment by JacksonBurns** [comment]
FWIW I also just attempted to use the binaries instead on a fresh ubuntu 22 and now have rms error in the same manner as #2290

**Comment by Laxzal** [comment]
Hey @JacksonBurns, if you are trying to install RMG-Py, then do the following once you have done the command mamba env create -f environment.yml and prior to the rms compile

#Prior to activation of the environment, the LD_LIBRARY_PATH needs to be set as an environmnet variable when rmg_env is activated.
#This exporting and unsetting will solve the RMS installation during the Julia compile and also when the environment is deactivated
#the original LD_LIBRARY_PATH is set.
echo "export OLD_LD_LIBRARY_PATH=${LD_LIBRARY_PATH}" >> $CONDA_BASE/envs/rmg_env/etc/conda/activate.d/env_vars.sh
echo "export LD_LIBRARY_PATH=$CONDA_BASE/envs/rmg_env/lib:$LD_LIBRARY_PATH" >> $CONDA_BASE/envs/rmg_env/etc/conda/activate.d/env_vars.sh
echo "export LD_LIBRARY_PATH=${OLD_LD_LIBRARY_PATH}" >> $CONDA_BASE/envs/rmg_env/etc/conda/deactivate.d/env_vars.sh
echo "unset OLD_LD_LIBRARY_PATH" >> $CONDA_BASE/envs/rmg_env/etc/conda/deactivate.d/env_vars.sh

**Comment by JacksonBurns** [comment]
Thanks @Laxzal - this fixed my issue on Ubuntu. With the totally fresh install I didn't have a `LD_LIBRARY_PATH` variable, so I just added it to my `.profile`.
If this is not part of a larger issue, I propose we revise the developer installation instructions.

**Comment by JacksonBurns** [comment]
This bug was ultimately resolved by bumping to Julia 1.8.5
